src/app/libraries/csrf_lib.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+class csrf_lib extends Library {
+
+    public function __construct() {
+        if (session_status() == PHP_SESSION_NONE) {
+            session_start();
+        }
+    }
+    /**
+     * Generates a new CSRF token
+     * @return string Base64 encoded token
+     */
+    public static function getNewCSRFToken() {
+        return base64_encode(openssl_random_pseudo_bytes(12));
+    }
+
+    /**
+     * Generates a new CSRF token and sets it as the current token
+     * @return string The token
+     */
+    public function setNewCSRFToken() {
```

Comment:
The name is slightly not so much intuitive. Anything better in mind? I thought getNewToken, but thet is already being used.